jx_sqlite/query_table.py

```diff
@@ -276,7 +281,10 @@ def query(self, query):
                 row = [None for _ in column_names]
                 for s in index_to_columns.values():
                     if s.push_child == ".":
-                        row[s.push_column] = s.pull(d)
+                        if not row[s.push_column]:                        
+                            row[s.push_column] = s.pull(d)
+                        else:
+                            row[s.push_column] += s.pull(d)
```

Comment:
As you suggested, I created the query to return the count of the relevant columns and then tried to add them as the output result. But on formatting, got the error saying:
`ERROR: unsupported operand type(s) for +=: 'Data' and 'int'`
so I `unwrap`'ed the `Data` type in those checks to get back the number, not `Data`.